### ISSUE TITLE
autoFill true causing whole website stuck

### ISSUE THREAD

**Issue by himanshusatija-cred** [justin-chu/react-fast-marquee#66]
If I'm using autoFill true, it's making whole website unresponsive.

**Comment by nathanielCherian** [comment]
Ran into this same issue, I believe it can be fixed if you wrap the Marquee in a div and bound its width to say 500px. The website crashes because it keeps on spawning more elements to fill the page.

**Comment by mezotv** [comment]
> Ran into this same issue, I believe it can be fixed if you wrap the Marquee in a div and bound its width to say 500px. The website crashes because it keeps on spawning more elements to fill the page.

Tried this and it still errors our a lot and crashes the site? Do you think @justin-chu will fix this?

**Comment by oguzhannftsiblings** [comment]
Same issue here. Fixed it by giving static width to it.

**Comment by lnicepei** [comment]
Had same issue, solved by @nathanielCherian's answer

**Comment by 6lyxt** [comment]
i will bump this

**Comment by BanShan-Alec** [comment]
Same issus，I hope the author can fix it or give tips in the documentation.

**Comment by ahbyass** [comment]
Wrapping the `<Marque />` in a grid with 1 column of 1fr width resolves the issue.

**Comment by mezotv** [comment]
> Wrapping the `<Marque />` in a grid with 1 column of 1fr width resolves the issue.

Seems like some hacky solution instead of actually fixing the issue.

**Comment by ahbyass** [comment]
> > Wrapping the `<Marque />` in a grid with 1 column of 1fr width resolves the issue.
> 
> Seems like some hacky solution instead of actually fixing the issue.

Agreed, but not my project, I passed along as a more flexible hack as you don't need to have it as a fixed width. There is a similar issue with Swiper.js, where width gets calculated to Infinity without fixed parent widths with different behaviours for flex and grid parents and the units used to define parent column width eg fr or % . Given the extensive discussion of this issue on swiper and the the agreed fix for that is this suggested hack, it may likely be the fix that is need internally with this project.
